TODO.md

```diff
@@ -0,0 +1,97 @@
+* additional Replacers should be used where appropriate. 
+
+* The following ngrok config options are not yet surfaced.
+
+```golang
+
+// https://github.com/ngrok/ngrok-go/blob/main/session.go
+// WithProxyURL configures the session to connect to ngrok through an outbound
+// HTTP or SOCKS5 proxy. This parameter is ignored if you override the dialer
+// with [WithDialer].
+//
+// See the [proxy url paramter in the ngrok docs] for additional details.
+//
+// [proxy url paramter in the ngrok docs]: https://ngrok.com/docs/ngrok-agent/config#proxy_url
+func WithProxyURL(url *url.URL) ConnectOption
```

Comment:
I have an idea for this one which I've implemented on a personal project:
starting with `type JsonURL string` with a  `UnmarshalJSON` method that validates the parsing, i.e. calling `url.Parse` is without error. It can be tackled later.